heartbeat/docs/getting-started.asciidoc

```diff
@@ -1,5 +1,5 @@
 [[heartbeat-getting-started]]
-== Getting Started With Heartbeat
+== Get started With Heartbeat
 
```

Comment:
Heartbeat is missing the title abbreviation for Get Started that the other Beats have:
```suggestion

++++
<titleabbrev>Get started</titleabbrev>
++++
```